workflow.txt

```diff
@@ -1,10 +1,10 @@
-git clone
+fork on github
 git checkout -b my-solution
-git commit -m ""
-git fetch --all
-git add .
 git pull upstream master
-git merge my-solution
-fork on github
-git checkout master
+git add .
+git commit -m ""
```

Comment:
after committing onto our my-solution branch we can checkout master
`git checkout master`
now is a good time to pull from upstream master since other people might have pushed changes while we were working in our branch
`git pull upstream master`
next we'll merge our branch into master and finally push
`git merge my-solution`
`git push origin my-solution`